virttest/qemu_vm.py

```diff
@@ -142,6 +142,7 @@ def __init__(self, name, params, root_dir, address_cache, state=None):
             self.process = None
             self.serial_ports = []
             self.serial_console_log = None
+            self.serial_session_device = None
             self.virtio_console = None
```

Comment:
We still have `self.virtio_ports` now, but  function `create_virtio_console` be totally removed.

Reply:
@yanan-fu You mean `self.virtio_console`?

Reply:
@vivianQizhu  we have `self.virtio_ports`,  but no `self.virtio_console` as function `create_virtio_console`  be removed.   `_create_serial_console` is only for `serials[0]`

Reply:
As we discussed,  user can only connect to one console (the first serial) at the same time, use common `_create_serial_console`. So remove ` self.virtio_console = None` here is ok.